docs/source/coupling_traffic/installation.rst

```diff
@@ -0,0 +1,79 @@
+==========================
+SUMO-JuPedSim Installation
+==========================
+
+For the latest release verion of *SUMO* installer for Windows and macOS are available `here <https://sumo.dlr.de/docs/Installing/index.html>`__.
+However, we recommend installing the latest development version (nightly snapshots) of *SUMO* so you can benefit from the most recent developments.
+The necessary steps for Windows and macOS are explained below.
+
+Windows Installer
+=================
+
+Go to this `website <https://sumo.dlr.de/docs/Downloads.php#nightly_snapshots>`__ and download the installer provided by *Windows installer with all extras*. 
+Follow the instructions of the installer – you should be ready to get started with the tutorials.
+
+
+Build on macOS
```

Comment:
why not use the pkg-file? 
because before ou write:
"*SUMO* installer for Windows and macOS are available"

Reply:
This section  is a little confusing. Why should I install stuff when I have to build it later? Or what exactly am I supposed to do? Install or build? or both?